externs/shaka/player.js

```diff
@@ -858,6 +859,13 @@ shaka.extern.DashManifestConfiguration;
  *   format this value.
  *   <i>Defaults to
  *   <code>'video/mp2t; codecs="avc1.42E01E, mp4a.40.2"'</code>.</i>
+ * @property {boolean} useSafariBehaviorForLive
+ *   The spec says nothing much about seeking in live content, but Safari's
+ *   built-in HLS implementation does not allow it.  Therefore we will set
```

Comment:
The wording in the second sentence here is confusing. It isn't clear it's describing the effect of this configuration value.
I would explicitly describe it in terms of what happens if it's set to true, vs if it's set to false. Something like:

> If this is true, playback will set the availability window to the presentation delay. The player will be able to buffer ahead three segments, but the seek window will be zero-sized, to be consistent with Safari.
> If this is false, the seek window will be the entire duration.

Reply:
Done